libraries/Microsoft.Bot.Connector/AttachmentsEx.cs

```diff
@@ -32,21 +32,19 @@ public partial class Attachments
         /// <param name="attachmentId">id of the attachment.</param>
         /// <param name="viewId">default is "original".</param>
         /// <returns>uri.</returns>
-#pragma warning disable CA1055 // Uri return values should not be strings (we can't change this without breaking binary compat)
-        public string GetAttachmentUri(string attachmentId, string viewId = "original")
-#pragma warning restore CA1055 // Uri return values should not be strings
+        public Uri GetAttachmentUri(string attachmentId, string viewId = "original")
```

Comment:
I was curious about how does he know that it has to return a Uri, and found this.
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1055#cause